go.mod

```diff
@@ -3,79 +3,87 @@ module github.com/fluent/fluent-operator
 go 1.17
```

Comment:
Maybe we can change this to `go 1.19` as well

Reply:
This should be a version error of `code-generate`